open-liberty/content.md

```diff
@@ -99,18 +106,31 @@ For greater flexibility over configuration, it is possible to mount an entire se
 ```console
 $ docker run -d -p 80:9080 \
   -v /tmp/DefaultServletEngine:/config \
-  %%IMAGE%%:webProfile8
+  %%IMAGE%%:full
 ```
 
-# Using `springBoot` images
+# Using Spring Boot with `kernel` images
 
-The `springBoot` images introduce capabilities specific to the support of Spring Boot applications, including the `springBootUtility` used to separate Spring Boot applications into thin applications and dependency library caches. To elaborate these capabilities this section assumes the standalone Spring Boot 2.0.x application `hellospringboot.jar` exists in the `/tmp` directory.
+The `full` images introduce capabilities specific to the support of all Liberty
+features, including Spring Boot applications. This image thus includes the
+`springBootUtility` used to separate Spring Boot applications into thin
+applications and dependency library caches. To get these same capabilities
+without including features you are not using, build instead on top of `kernel` images
+and run configure.sh for your server.xml.
```

Comment:
after "server.xml" I would add something like ", ensuring it enables either the `springBoot-1.5` or `springBoot-2.0` feature."